example/app-with-native-dependencies.dockerfile

```diff
@@ -13,7 +13,7 @@ RUN bash $SCRIPTS_FOLDER/build-meteor-bundle.sh
 
 
 # Use the specific version of Node expected by your Meteor release, per https://docs.meteor.com/changelog.html; this is expected for Meteor 2.7.2
-FROM node:14.18.3-alpine
+FROM node:14.19.1-alpine
```

Comment:
Where did you get this? I’m not seeing it in the Meteor changelog (though it really should be). I’d like to update the URL in the comment to a better source for where this version can be retrieved.

Also this should really be its own PR, I think. Thanks for updating this.

Reply:
This happened in 2.7.0

v2.7, 2022-03-24

Highlights
Bump node version to 14.19.1

I only included it because it was part of my debugging of this problem.